lib/services/importMap/utils.js

```diff
@@ -177,6 +220,9 @@ module.exports = (config) => {
       const rooms = await getMapFromFile(filePath)
       await loadRooms(config, filePath, rooms)
       return logFinish()
+    },
+    async exportMap () {
```

Comment:
This signature doesn't match the help text

Reply:
Oh right, I took that out because it felt easier to just dump into the asset directory. I'll remove that.